next.config.js

```diff
@@ -35,4 +35,7 @@ module.exports = withMDX({
 
     return config
   },
+  env:{
+    YOUTUBE_TOKEN: process.env.YOUTUBE_TOKEN
+  }
```

Comment:
I don't remember if we still need that token on browser side as we fetch videos on workflow side.

```suggestion
```